src/main/java/frc/robot/Robot.java

```diff
@@ -88,8 +98,7 @@ public void disabledPeriodic() {
     public void autonomousInit() {
         drivetrain.resetEncoders();
         drivetrain.resetPigeon();
-        new GyroAutonomous(drivetrain).schedule();
-//        new OneCargoAutonomous().schedule();
+        autoCommand.schedule();
```

Comment:
change this to `new GyroAutonomous(drivetrain).schedule()` and under it `//new OneCargoAutonomous().schedule();`